packages/server/src/responder/component.ts

```diff
@@ -21,37 +21,37 @@ export enum ResponderStateKind {
 export type PendingResponseState = {
     appointmentId: string;
     kind: ResponderStateKind.Pending;
-    identifier: PisaTransactionIdentifier;
+    identifier: PisaTransactionIdentifierSerialisation;
 };
 export type MinedResponseState = {
     appointmentId: string;
     kind: ResponderStateKind.Mined;
-    identifier: PisaTransactionIdentifier;
+    identifier: PisaTransactionIdentifierSerialisation;
     blockMined: number;
     nonce: number;
 };
-export type ResponderAppointmentAnchorState = PendingResponseState | MinedResponseState;
+export type ResponderAppointmentAnchorState = PlainObject & (PendingResponseState | MinedResponseState);
```

Comment:
Do you need this?

Reply:
Looks, like I don't, I added it on a previous iteration of the type system. Removed.